internal/distro/rhel85/distro.go

```diff
@@ -353,6 +355,15 @@ func (t *imageType) sources(packages []rpmmd.PackageSpec, ostreeCommits []ostree
 	return sources
 }
 
+func checkMountpoint(mountpoint string) bool {
```

Comment:
We should check for prefixes, so `/var/log` should be accepted, but not `/variable`.

Reply:
Thanks, been thinking how to do that and regex is probably the way to go here

Reply:
I wonder if Golang has any nice file path helper library?

Reply:
Might be an overkill: https://github.com/spf13/afero

Reply:
Ahhh I think golang path will be enough for this actually. Just writing a couple of tests.